liferay-platform-experience/liferay-portal
# LPD 31478 - Theme CSS client extension is not loaded on Page with Arabic locale

Ticket: https://liferay.atlassian.net/browse/LPD-31478 (LPP 54641)

## Motivation
Currently, when adding a themeCss client extension, only **main.css** and **clay.css** ( which are LTR by default ) are possible to be provided. RTL versions are automatically generated, but its broken when there are custom css properties

## Proposed Solution
This changes enable the users to provide pre-built RTL versions of **main.css** and **clay.css** in case they need.

## Steps to Verify
_Reproduction Steps:

1. Start 2024.q1.2 and deploy the attached CSS sample: "liferay-sample-theme-css-1.zip" liferay-sample-theme-css-1.zip 
2. Go to Site Builder --> Pages, click on the kebab menu on the upper right, select "Configuration". Under the "Design" scroll down until you find "Theme CSS", and click on the icon to select the CSS, and then save.
3. Go to the Home page, edit it, add a Card fragment, make sure that the Card title color is Blue, and Publish the Page.
4. On the Page URL add the Arabic locale /ar/, and enter.
5. Hard refresh the page by clicking CTRL+SHIFT+R and check the Card Title Color.

Actual Result: The Card title color does not remain blue, the color is reset to the default. The CSS is not inserted.

Expected Result: The Card title color should remain Blue._

## Screenshots/videos (if applies)
Attached on ticket

## Alternative Solutions that were discarded (if applies)
Are there other solutions that you considered or tried? If so, tell us why you decided to discard them.

## General checklist (just for reference)

- [ x ] I have considered breaking this PR into smaller PRs if the amount of changed code is too large.
- [ x ] I have performed a self-review of my own code following Liferay's code conventions.
- [ x ] I have added at least one test that proves my fix is effective or that my feature works.
